pkg/api/types.go

```diff
@@ -65,6 +85,19 @@ type Config struct {
 	// Tag is a result image tag name.
 	Tag string
 
+	// BuilderPullPolicy specifies when to pull the builder image
+	BuilderPullPolicy PullPolicy
+
+	// PreviousImagePullPolicy specifies when to pull the previously build image
+	// when doing incremental build
+	PreviousImagePullPolicy PullPolicy
+
+	// ForcePull defines if the builder image should be always pulled or not.
+	// This is now deprecated by BuilderPullPolicy and will be removed soon.
+	// Setting this to 'true' equals setting BuilderPullPolicy to 'PullAlways'.
+	// Setting this to 'false' equals setting BuilderPullPolicy to 'PullIfNotPresent'
+	ForcePull bool
```

Comment:
This is now deprecated by BuilderPullPolicy and will be removed soon. 

this is not enough?